packages/help-center/src/components/help-center-container.tsx

```diff
@@ -48,38 +48,25 @@ const HelpCenterContainer: React.FC< Container > = ( {
 	}, [] );
 
 	const nodeRef = useRef< HTMLDivElement >( null );
-
 	const { setIsMinimized } = useDispatch( HELP_CENTER_STORE );
-	const [ isVisible, setIsVisible ] = useState( true );
 	const isMobile = useMobileBreakpoint();
 	const classNames = clsx( 'help-center__container', isMobile ? 'is-mobile' : 'is-desktop', {
 		'is-minimized': isMinimized,
 	} );
 
 	const onDismiss = useCallback( () => {
-		setIsVisible( false );
+		handleClose();
 		recordTracksEvent( `calypso_inlinehelp_close` );
-	}, [ setIsVisible ] );
-
-	const toggleVisible = () => {
-		if ( ! isVisible ) {
-			handleClose();
-			// after calling handleClose, reset the visibility state to default
-			setIsVisible( true );
-		}
-	};
+	}, [ handleClose ] );
 
 	const animationProps = {
 		style: {
-			animation: isMobile
-				? `${ isVisible ? 'fadeIn' : 'fadeOut' } .25s ease-out`
-				: `${ isVisible ? 'slideIn' : 'fadeOut' } .25s ease-out`,
+			animation: isMobile ? 'fadeIn .25s ease-out' : 'slideIn .25s ease-out',
```

Comment:
We have `is-mobile` class in the SASS file, we can move this animation there since it doesn't have any JS anymore.